lib/ask-stack-result-view.coffee

```diff
@@ -198,7 +215,17 @@ class AskStackResultView extends ScrollView
           atom.workspace.activatePreviousPane()
           # editor = atom.workspace.activePaneItem
           editor = atom.workspace.getActivePaneItem()
-          editor.insertText(code)
+          if qa == true
+            editor.insertText("Insert from Stack Overflow", {select: true})
```

Comment:
I don't think that's required as the source and author link clearly shows it's from Stack Overflow.